CMakeLists.txt

```diff
@@ -118,6 +118,15 @@ set(LIBDDWAF_SOURCE
     ${libddwaf_SOURCE_DIR}/src/transformer/js_decode.cpp
     ${libddwaf_SOURCE_DIR}/src/transformer/common/utf8.cpp
     ${libddwaf_SOURCE_DIR}/src/libcxx-compat/monotonic_buffer_resource.cpp
+    ${libddwaf_SOURCE_DIR}/src/vendor/fmt/format.cc
```

Comment:
why not std::fmt?

Reply:
I tried but it's not widely supported on all of our builds, support seems to be available in gcc 13, libcxx-14 (although it seems to be experimental even in 16) :
- Mac OS still uses llvm-13
- Our older docker builds use llvm-12
- MinGW builds with gcc-10

We could conditionally use one or the other but only windows compiles without adding extra libraries (e.g. libcxx-experimental for llvm-16).